### PR TITLE
AT_01.02_030 | <Header> User is able to select a single result if there are multiple matches in the search dropdown

### PR DESCRIPTION
**Pull request by milladiz84**
https://trello.com/c/IQwJ9pFv/1241-at0102030-header-user-is-able-to-select-a-single-result-if-there-are-multiple-matches-in-the-search-dropdown

### CODE REVIEW

File: cypress/e2e/headerSearchBox.cy.js [hunk 1]
```diff
@@ -167,6 +167,20 @@ describe('Header Search Box', () => {
...
+    cy.get('input#search-box').type(headers.dataLetter).realHover();
+    let selectWord;
+    cy.get('#search-box-completion li:not([style="display: none;"])').each(($el, index, $list) => {
+      const dropDown = $el.text().trim();
+      if (index === 0) {
+        selectWord = dropDown;
+        cy.wrap($el).click()
+      }
+    }).then(() => {
+      cy.log(selectWord)
```
Comment (Rom-T) [inline cypress/e2e/headerSearchBox.cy.js:182]: Please delete `log`

Comment (milladiz84) [reply cypress/e2e/headerSearchBox.cy.js]: thanks, I did)

File: cypress/e2e/headerSearchBox.cy.js [hunk 2]
```diff
@@ -167,6 +167,19 @@ describe('Header Search Box', () => {
...
       cy.get('div.error').should('have.text', headers.textNothing)
     }
-
   })
-});
+
+  it('AT_01.02_030 | Select a single result if there are multiple matches in the search dropdown', () => {
+    cy.get('input#search-box').type(headers.dataLetter).realHover();
+    let selectWord;
+    cy.get('#search-box-completion li:not([style="display: none;"])').each(($el, index, $list) => {
```
Comment (Rom-T) [inline cypress/e2e/headerSearchBox.cy.js:175]: And `$list` like is not used anywhere.

**Review by Rom-T — APPROVED** [review]
LGTM

File: cypress/e2e/headerSearchBox.cy.js (hunk 2 above)
Comment (milladiz84) [reply cypress/e2e/headerSearchBox.cy.js]: I agree, this param don't need here, deleted

**Review by lanasussman — APPROVED** [review]

**Pull request merged**